src/test/scripts/helm_install.sh

```diff
@@ -47,6 +47,12 @@ helm install -n "${TARGET_NAMESPACE}" --wait \
    --version "$POSTGRES_CHART_VERSION" \
    bitnami/postgresql > $LOG_DOWNLOAD_DIR/helm_install_log.txt
 
+if [ -n "$2" ]
+then
+  kubectl cp $2 $TARGET_NAMESPACE/$POSTGRES_RELEASE_NAME-0:/tmp/db-init-script.sql
+  kubectl exec --stdin --tty $POSTGRES_RELEASE_NAME-0 -- /bin/bash -c "psql postgresql://$PRODUCT_NAME:$PRODUCT_NAME@localhost:5432/$PRODUCT_NAME -f /tmp/db-init-script.sql"
```

Comment:
Have we tested to make sure that if the db script fails, the `kubectl exec` will also fail? It's important that the install script halts at that point.

Reply:
`-c "exit 1"` halts installation script and connecting with wrong username also halts the install script. Any other returned value than 0 stops the install script.

`+ kubectl exec bkwiatek-jira-pgsql-0 -- /bin/bash -c 'psql postgresql://abc:jira@localhost:5432/jira -f /tmp/db-init-script.sql'
psql: FATAL:  password authentication failed for user "abc"
command terminated with exit code 2`